### PR TITLE
add details for CDT case study

### PR DESCRIPTION
**Pull request by jc2820**
First Case study up. Follows the layout we chose on miro before. LEt me know what you think about language/voice etc.

Add any reflections that I forgot!

### CODE REVIEW

File: case-studies/CDT.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Introduction
+
+CDT tested the idea that more than one charity could collaboratively learn the basics of digital service design and product management by identifying a common problem area for those charities to develop into a shareable solution. 
+
+Through design sprint discovery and definition workshops, two charities worked together with facilitators from Founders and Coders to find a common problem area and find a mutually beneficial solution idea.
+
+Product management workshops introduced the roles and responsibilities of a product owner and learning exercises related to product management during development of a digital product.
+
+With a prototype created by Founders and Coders and usability tested by the charities, the final section of the programme involved building an MVP over three development sprints where the charity leads acted as product owners conducting usability testing and taking part in sprint review and planning.
```
Comment (Albadylic) [inline case-studies/CDT.md:16]:
```suggestion
With a prototype created by Founders and Coders and usability tested by the charities, the final section of the programme involved building an MVP over three development sprints where the charity leads acted as Product Owners conducting usability testing and taking part in sprint review and planning.
```

**Review by bethanyios — APPROVED** [review]
Excellent!
Nothing I would change at the moment - love the clarity, we should maintain that for all case studies.

File: case-studies/CDT.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Build
+* Product management exercises were repeated weekly while the MVP was being built
+* Towards the close of the programme there were playback meetings with all partners and a sprint retrospective to close (Stop, Go, Continue)
+
+---
+
+## Reflections
+
+* CDT showed that it is possible to find common ground in a small cohort of charity's problem areas, and work collaboratively to design and develop a solution.
```
Comment (Albadylic) [inline case-studies/CDT.md:64]:
```suggestion
* CDT showed that it is possible to find common ground in a small cohort of charites' problem areas, and work collaboratively to design and develop a solution.
```

**Review by Albadylic — APPROVED** [review]
I think this is excellent. Exactly the right tone and a good overview of what we did & what we learnt. 

Two small (pedantic) corrections suggested

**Pull request merged**